DockeriseRmap/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rocker/r-base:4.2.2
+
+USER  root
+
+RUN R -e "install.packages(c('tidyr', 'dplyr', 'data.table', 'stringr', 'Rsamtools', 'argparser', 'BiocManager'), dependencies=TRUE, repos='http://cran.rstudio.com/')"
+RUN apt-get update -y && apt-get install -y libcurl4-openssl-dev samtools gawk tabix bcftools
```

Comment:
apt-get should be first in the build process, also I'd expect to see purging of the apt lists.  pre-commit should be catching this, add this to pre-commit:

https://github.com/cynapse-ccri/cynapse-burge-pilot/blob/cabea1939cf1435f382d5ce3be2bb548e3ddd3fb/.pre-commit-config.yaml#L23-L27

Reply:
Done